page/about.html

```diff
@@ -106,11 +107,10 @@ <h2 class="heading-secondary u-center-text u-margin-bottom-medium">
       <section class="section-2">
         <div class="row flex-d-culumn">
           <h3 class="heading-tertiary u-center-text u-margin-bottom-medium">
-            See the past CC Global Summits
+            Come closer to our web agency
           </h3>
           <h2 class="heading-secondary u-center-text u-margin-bottom-medium">
-            Take a look at the last two CC Global Summits which took place in
-            Buenos Aires and in Warsaw.
+            website creation: from an experienced web agency
           </h2>
 
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove the `footer` part from `About` page and pay attention to your `text designs` 🙏🏼 

![image](https://github.com/ahmidou09/CreativeStudioCapstone/assets/63915024/ad3fe3f8-1045-458f-9e80-e3982719b182)